permissions/plugin-conjur-credentials.yml

```diff
@@ -10,4 +10,5 @@ developers:
 - "cyberark_bizdev"
 security:
   contacts:
-    email: "product_security@cyberark.com"
\ No newline at end of file
+    email: "product_security@cyberark.com"
```

Comment:
you may only have one
```suggestion
```

Reply:
Exactly :) I will deprecate soon the email completely, so better to just have the `jira` in this PR to move forward.